R/tmb-methods.R

```diff
@@ -103,6 +104,7 @@ vcov.mmrm_tmb <- function(object, complete = TRUE, ...) {
 #' # Variance-covariance matrix estimate for residuals:
 #' VarCorr(object)
 VarCorr.mmrm_tmb <- function(x, sigma = NA, ...) { # nolint
+  # nolint
```

Comment:
might not need it here if we have it above on 105, right?

Reply:
thx!